docs/index.rst

```diff
@@ -52,10 +52,9 @@ Plasma Parameters
 -----------------
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 2
```

Comment:
Did this have to do with why all the atomic stuff was flat instead of a pretty tree?

Reply:
I'm still tinkering with that part! It's a factor, but right now I think it's the fact that atomic/index isn't linked here while the subpages are. Tinkering with maxdepth should allow us to simply link that page and not worry about subpages. But I couldn't verify that earlier as I just now figured out how to run stuff locally with the RTD Sphinx theme.